tests/testthat/test_measures_net_emd.R

```diff
@@ -167,40 +167,67 @@ test_that("net_emd returns min_emd = 0 and min_offset = 0 when comparing any
 
   offset_lists <- replicate(num_hists, offsets, simplify = FALSE)
 
-  expect_self_net_emd_correct <- 
+  expect_self_net_emd_correct <-
     function(histogram, shift, method, return_details = FALSE) {
-    self_net_emd <- net_emd(histogram, shift_dhist(histogram, shift), 
+    self_net_emd <- net_emd(histogram, shift_dhist(histogram, shift),
                             method, return_details)
     expected <- list(net_emd = 0, min_emds = 0, min_offsets = shift)
     expect_equal(self_net_emd, expected)
   }
 
   purrr::walk2(rand_dhists, offset_lists, function(dhist, offsets) {
     purrr::walk(offsets, function(offset){
-      expect_self_net_emd_correct(dhist, offset, "optimise", 
+      expect_self_net_emd_correct(dhist, offset, "optimise",
                                   return_details = TRUE)
     })
   })
-  
+
   purrr::walk2(rand_dhists, offset_lists, function(dhist, offsets) {
     purrr::walk(offsets, function(offset){
-      expect_self_net_emd_correct(dhist, offset, "exhaustive", 
+      expect_self_net_emd_correct(dhist, offset, "exhaustive",
                                   return_details = TRUE)
     })
   })
 })
 
```

Comment:
New tests inserted below. All other changes to this file are line endings and indentation.